examples/support-bundle/e2e.yaml

```diff
@@ -11,10 +11,10 @@ spec:
         data: "5"
     - runPod:
         collectorName: "static-hi"
-        podSpec: 
+        podSpec:
           containers:
           - name: static-hi
-            image: alpine:3.5
+            image: alpine:latest
```

Comment:
I do not think that is a good idea use latest
That can result in the tests starts to fail and we do not know why.

To ensure that the test are idempotent the best is to pinned the versions

Reply:
> I do not think that is a good idea use latest That can result in the tests starts to fail and we do not know why.
> 
> To ensure that the test are idempotent the best is to pinned the versions

Changed to `alpine:3`